### PR TITLE
Add AMP for ResNet200_vd model

### PR DESCRIPTION
**Pull request by pangyoki**
Train ResNet200_vd with amp

### CODE REVIEW

File: PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -ex
+
+#Training details
+export FLAGS_conv_workspace_size_limit=4000 #MB
+export FLAGS_cudnn_exhaustive_search=1
+export FLAGS_cudnn_batchnorm_spatial_persistent=1
+
+DATA_DIR="Your image dataset path, e.g. ./data/ILSVRC2012/"
+DATA_FORMAT="NCHW"
```
Comment (zhangting2020) [inline PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh]: 这个脚本是AMP的训练配置，所以直接设置DATA_FORMAT=“NHWC”吧

File: PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+USE_AMP=true #whether to use amp
+USE_DALI=true
+USE_ADDTO=true
+if ${USE_AMP} ;then
+    DATA_FORMAT="NHWC"
```
Comment (zhangting2020) [inline PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh]: 前面直接设置为NHWC，这里可以删除了，可以参考ResNet50_fp16的配置
https://github.com/PaddlePaddle/models/blob/cd03506decae14b6b983640c4637ff6d8bc5639b/PaddleCV/image_classification/scripts/train/ResNet50_fp16.sh#L10-L15

File: PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh (hunk 1 above)
Comment (pangyoki) [reply PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh]: done

File: PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh (hunk 2 above)
Comment (pangyoki) [reply PaddleCV/image_classification/scripts/train/ResNet200_vd_fp16.sh]: done

**Review by zhangting2020 — APPROVED** [review]
LGTM

**Pull request merged**